src/ripple/protocol/STBitString.h

```diff
@@ -25,7 +25,7 @@
 
 namespace ripple {
 
-template <std::size_t Bits>
+template <int Bits>
```

Comment:
I suggest adding a comment to the class declaration explaining the reason that this template parameter is `int` so that nobody comes along later and changes it back thinking they're doing the right thing.